server/prisma/seed.ts

```diff
@@ -5,5 +5,7 @@ const prisma = new PrismaClient();
 (async () => {
   await prisma.user.deleteMany();
 
-  await prisma.user.create({ data: { email: 'krzysztof@jarzyna.com' } });
+  await prisma.user.create({
+    data: { id: '1', username: 'Krzysiu', nickname: 'Krzychu007', email: 'krzy@szt.of', avatar: 'emptyhttps', toWorkDistance: 2.5 },
```

Comment:
czy przypadkiem prisma sama nie przypisuje wartosci do pola id?

Reply:
![image](https://user-images.githubusercontent.com/35918401/163264693-32482c44-8985-4b33-ab7c-55e162bd2514.png)

Reply:
nie, dlatego że ID bierzemy od googla

Reply:
No właśnie ale to branie Id z googla nam trochę blokuje rozbudowę o innych providerów bo np twitter może nadać to samo id innemu użytkownikowi, wystawiamy się na ryzyko kolizji (niewielkie ale jednak). Ten sam problem jeżeli chcielibyśmy mieć własną rejestracje to jakie id tam wpisać?

Ja bym zrobił tak że a id jest tak jak było: `id     String  @id @default(uuid())` a na to można zrobić pole googleId (jeżeli w ogóle potrzebujemy przechowywać id - maile powinny być unikalne)

@matizaj id się samo generuje bo w schemie był default zapisany

Reply:
zmieniłem, id generuje się automatycznie, to czy użytkownik już istnieje w bazie danych jest wyszukiwane za pomocą emaila